pandas/tests/tools/test_to_datetime.py

```diff
@@ -2263,73 +2206,23 @@ def test_to_datetime_infer_datetime_format_consistent_format(
         s_as_dt_strings = ser.apply(lambda x: x.strftime(test_format))
 
         with_format = to_datetime(s_as_dt_strings, format=test_format, cache=cache)
-        no_infer = to_datetime(
-            s_as_dt_strings, infer_datetime_format=False, cache=cache
-        )
-        yes_infer = to_datetime(
-            s_as_dt_strings, infer_datetime_format=True, cache=cache
-        )
+        no_infer = to_datetime(s_as_dt_strings, cache=cache)
+        yes_infer = to_datetime(s_as_dt_strings, cache=cache)
 
         # Whether the format is explicitly passed, it is inferred, or
         # it is not inferred, the results should all be the same
         tm.assert_series_equal(with_format, no_infer)
         tm.assert_series_equal(no_infer, yes_infer)
 
     @pytest.mark.parametrize(
-        "data",
-        [
-            ["01/01/2011 00:00:00", "01-02-2011 00:00:00", "2011-01-03T00:00:00"],
-            ["Jan/01/2011", "Feb/01/2011", "Mar/01/2011"],
-        ],
+        "tz_name, offset, warning",
+        [("UTC", 0, None), ("UTC-3", 180, UserWarning), ("UTC+3", -180, UserWarning)],
     )
-    def test_to_datetime_infer_datetime_format_inconsistent_format(self, cache, data):
-        ser = Series(np.array(data))
-
-        # When the format is inconsistent, infer_datetime_format should just
-        # fallback to the default parsing
-        tm.assert_series_equal(
-            to_datetime(ser, infer_datetime_format=False, cache=cache),
-            to_datetime(ser, infer_datetime_format=True, cache=cache),
-        )
-
-    def test_to_datetime_infer_datetime_format_series_with_nans(self, cache):
-        ser = Series(
-            np.array(
-                ["01/01/2011 00:00:00", np.nan, "01/03/2011 00:00:00", np.nan],
-                dtype=object,
-            )
-        )
-        tm.assert_series_equal(
-            to_datetime(ser, infer_datetime_format=False, cache=cache),
-            to_datetime(ser, infer_datetime_format=True, cache=cache),
-        )
-
-    def test_to_datetime_infer_datetime_format_series_start_with_nans(self, cache):
-        ser = Series(
-            np.array(
-                [
-                    np.nan,
-                    np.nan,
-                    "01/01/2011 00:00:00",
-                    "01/02/2011 00:00:00",
-                    "01/03/2011 00:00:00",
-                ],
-                dtype=object,
-            )
-        )
-
-        tm.assert_series_equal(
-            to_datetime(ser, infer_datetime_format=False, cache=cache),
-            to_datetime(ser, infer_datetime_format=True, cache=cache),
-        )
```

Comment:
Maybe keep the above 3 tests (suitably modified to remove `infer_datatime_format`), but check that we raise because of the inconsistent warnings